iocore/eventsystem/I_VConnection.h

```diff
@@ -116,13 +116,16 @@
 //
 //////////////////////////////////////////////////////////////////////////////
 
-/** Used in VConnection::shutdown(). */
-enum ShutdownHowTo_t {
-  IO_SHUTDOWN_READ = 0,
-  IO_SHUTDOWN_WRITE,
-  IO_SHUTDOWN_READWRITE,
+enum class IOShutdown {
+  READ      = SHUT_RD,
+  WRITE     = SHUT_WR,
+  READWRITE = SHUT_RDWR,
 };
 
+constexpr IOShutdown IO_SHUTDOWN_READ      = IOShutdown::READ;
+constexpr IOShutdown IO_SHUTDOWN_WRITE     = IOShutdown::WRITE;
+constexpr IOShutdown IO_SHUTDOWN_READWRITE = IOShutdown::READWRITE;
```

Comment:
Why don't you use IOShutdown::READ|WRITE|READWRITE directly?  For maintainability of the code, now I need to lookup up IO_SHUTDOWN_READ to figure out it is a IOShutdown::READ